day_10/task_2/tests/pipe_maze_tests.cpp

```diff
@@ -129,6 +129,45 @@ std::vector<MazeScopeParams> GenerateMazeScopeParams()
     };
 }
 
+INSTANTIATE_TEST_SUITE_P(IsInMazeScope,
+                         MazeScopeTests,
+                         testing::ValuesIn(GenerateMazeScopeParams()));
+
+
+TEST_P(MazeScopeTests, GivenMazePoint_WhenColorNeighborForPoint_ExpectColorEightPoint)
+{
+    // const PipeMaze pipe_maze = {
+    // "- L | F 7",
+    // "7 S - 7 |",
+    // "L | 7 | |",
+    //  "- L - J |",
+    //  "L | -J F"};
+
+    auto [color_right, color_top_right, color_down_right, color_top, color_down,
+          color_left, color_top_left, color_down_left] = GetParam();
```

Comment:
Please change the order here to be clockwise ordered or column wise( from top to bottom, column by column from left to right or from right to left)

and please bear in mind to change accordingly the order of ASSERTS_EQ as well

Reply:
done